jjb/mkck/mkck-trigger.yaml

```diff
@@ -276,7 +276,7 @@
             default: 'makecheck-opensuse-15.0-x86_64'
         - string:
             name: leap15_1
-            default: 'makecheck-opensuse-15.1-x86_64'
```

Comment:
Instead of this I would add a link makecheck-opensuse-15.1 in filesystem, but I prefer to rename the conf to the file with x86_64 suffix, and then link it as I did for 15.0.

Reply:
A new symlink in the filesystem is much better, yes. I did send a follow-up commit :)